OMCompiler/SimulationRuntime/OMSICpp/CMake/FindSuiteSparse.cmake

```diff
@@ -13,10 +13,10 @@ SET(SUITESPARSE_UMFPACK_FOUND false)
 SET(SUITESPARSE_UMFPACK_USE_OMC false)
 
 SET(SUITESPARSE_UMFPACK_INCLUDE_DIR_OMC "")
-MESSAGE(STATUS "Searching for umfpack.h of OpenModelica in ${CMAKE_INSTALL_PREFIX}/include/omc/c/suitesparse/Include")
-FIND_FILE(UMFPACK_H_OMC umfpack.h HINTS "${CMAKE_INSTALL_PREFIX}/include/omc/c/suitesparse/Include" NO_DEFAULT_PATH)
+MESSAGE(STATUS "Searching for umfpack.h of OpenModelica in ${CMAKE_INSTALL_PREFIX}/include/omc/c/suitesparse")
```

Comment:
Why do the include files for suitesparse go in `omc/c` btw? They belong where the other 3rdParty includes are. This together with the now-fixed `Include` (capital I) directory was confusing.  

It is cleaner and expected to have the install folder structure to match the source code folder structure if possible.

Reply:
This is also (maybe) why we have problems for dependents of suitesparse.  Include files are included from the install/build folder instead of the 3rdParty folder. These means suitesparse need to be built and installed before dependencies (which only need the headers) can be built.

Reply:
>Why do the include files for suitesparse go in omc/c btw? 

This is indeed a good question. I left it as it was but I think all 3rdParty includes should be independent of the C or Cpp runtimes. I can change that in a new PR or we do it when we do some more CMake magic for 3rdParty.

Reply:
Yes. Please change it if you can. I was going to change it but had to deal with the other CMake stuff first. 

Includes should be fixed everywhere to be the form `#include suitesparse/...` and during **build** of omc we should use specify 3rdParty folder as include directory `-I .../3rdParty.` And for **simulation** we will use build/install directory's `include/` folder as an include directory `-I .../include/ `

Reply:
Some 3rdParty things are part of the source code FMUs. Not all, but some. I think suitesparse isn't needed though.

Reply:
At the moment not, but that might change soon. We already have CVODE in FMUs. Using klu in there wouldn't be much of a change.
I'll open a new PR to change linking with BLAS/LAPACK on Windows and an additional one to change the include directories.

Should this PR be merged as it is now?

Reply:
From my side you can merge it. This was just a suggestion to improve later. You can leave it for now since things work. We can plan on it later together with the other CMake changes.